src/Mod/Sketcher/Gui/CommandCreateGeo.cpp

```diff
@@ -1005,6 +1007,8 @@ class DSHandlerDefaultWidget: public DrawSketchGeometryHandler<PTool, SelectMode
                         break;
                 }
             }
+
+            prevCursorPosition = onSketchPos;//Register custor position after modifying it.

```

Comment:
This is dangerous. The implementation relies in that prevCursorPosition is the last position that came from the mouse. This is changed, at a given time, based on the widget state. The first call after the mouse position is registered will certainly give the wanted result. Subsequent calls (without moving the mouse), will only give the desired result if the second widget state cancels the previous modification. This cannot be guaranteed for every single implementation (a parameter was set then unset).